src/shapes/image.class.js

```diff
@@ -410,6 +410,7 @@
                                      imageMargins.height
                                     );
 
+      this._stroke(ctx);
```

Comment:
Image class is the only one with the internal _stroke method. We can call it directly from its render method